### DISCUSSION TITLE
How can I add a non-breaking space in a variables file?

### DISCUSSION THREAD

**Discussion by stefan-jung** [dita-ot/dita-ot#4096] [Q&A] [answered]
I would like to add a non-breaking space in a [variables file](https://www.dita-ot.org/dev/topics/plugin-addgeneratedtext.html). How can I do that?

**Comment by chrispy-snps** [comment, marked as answer]
@xephon2 - try using `&#160;` in your string value.

**Comment by stefan-jung** [comment]
Challenge for the day: Try to place this entity in this Arabic **`<variable>`** element:

```xml
<variable id="On the page"> بالصفحة <param ref-name="pagenum"/></variable>
```
